### PR TITLE
testing/miniflux: new aport

### PR DESCRIPTION
**Pull request by fguillot**
https://miniflux.app/
Miniflux is a minimalist and opinionated feed reader.

### CODE REVIEW

File: testing/miniflux/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Contributor: Frédéric Guillot <fred@miniflux.net>
+# Maintainer: Frédéric Guillot <fred@miniflux.net>
+pkgname=miniflux
+pkgver=2.0.16
+pkgrel=0
+pkgdesc="Minimalist and opionated feed reader"
+url="https://miniflux.app/"
+arch="all"
+license="Apache-2.0"
+depends=""
```
Comment (maxice8) [inline testing/miniflux/APKBUILD]: can be removed

File: testing/miniflux/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+makedepends="go"
+install="$pkgname.pre-install"
+subpackages="$pkgname-openrc $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/miniflux/miniflux/archive/${pkgver}.tar.gz
+	miniflux.initd
+	miniflux.confd
+	miniflux.conf
+	miniflux.logrotate
+	"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/miniflux/APKBUILD]: it is the default value can be removed

File: testing/miniflux/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+build() {
+	cd "$builddir"
+	make miniflux VERSION=$pkgver
+}
+
+check() {
+	cd "$builddir"
+	./miniflux -version > /dev/null
```
Comment (maxice8) [inline testing/miniflux/APKBUILD]: please use the upstream test suite or disable check with a comment explaining.

Comment (fguillot) [reply testing/miniflux/APKBUILD]: Fixed

File: testing/miniflux/APKBUILD (hunk 2 above)
Comment (fguillot) [reply testing/miniflux/APKBUILD]: Fixed

File: testing/miniflux/APKBUILD (hunk 1 above)
Comment (fguillot) [reply testing/miniflux/APKBUILD]: Done

**Pull request closed without merging**